test/metabase/models/action_test.clj

```diff
@@ -134,3 +134,17 @@
                Exception
                #"Actions must be made with models, not cards"
                (t2/update! Action action-id {:archived false}))))))))
+
+(deftest exclude-non-required-pks-for-create-implicit-actions-test
```

Comment:
The name of this test is too broad. It's only testing auto increment fields

Reply:
updated